packages/core/custom-branding/core-custom-branding-server/types.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { KibanaRequest } from '@kbn/core-http-server';
+import { CustomBranding } from '@kbn/core-custom-branding-common';
```

Comment:
NIT: `import type` (a few other occurrences in commits since last review but won't comment on each)